ui/raidboss/data/06-ew/raid/p1s.txt

```diff
@@ -0,0 +1,128 @@
+### P1S: Asphodelos: The First Circle (Savage)
+#
+# -it 'Erichthonios' -oc 'Erichthonios' -p "6629:14.1"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+# First flail set
+0.0 "--sync--" sync /Engage!/ window 0,1
+9.4 "--middle--" sync / 14:[^:]*:Erichthonios:6629:/ window 11,9
+14.1 "Heavy Hand" sync / 1[56]:[^:]*:Erichthonios:6629:/
+18.2 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+22.8 "Aetherial Shackles" sync / 1[56]:[^:]*:Erichthonios:6625:/
+31.9 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+41.7 "Chain Pain" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+57.7 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:65F[6789]:/
+
+# First tank mechanic
+72.1 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
+75.5 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+
+# Second flail set
+91.1 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:6(?:5F[6-F]|60[01]):/
+105.3 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+
+# Tiles 1
+118.5 "Intemperance" sync / 1[56]:[^:]*:Erichthonios:661E:/
+122.6 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+134.2 "Intemperate Torment" sync / 1[56]:[^:]*:Erichthonios:661F:/
+135.3 "First Element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+141.4 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+146.4 "Second element" sync / 1[56]:[^:]*:Erichthonios:662[123]:/
+151.6 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+157.3 "Third element" sync / 1[56]:[^:]*:Erichthonios:662[12]:/
+
+# Second tank mechanic
+167.9 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
+171.1 "True Flare/True Holy" sync / 1[56]:[^:]*:Erichthonios:661[01]:/
+
+# Cells 1
+188.2 "Shining Cells" sync / 1[56]:[^:]*:Erichthonios:6616:/
+194.5 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+208.6 "Powerful Fire/Powerful Light + Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:661[AB]:/
```

Comment:
Can these be separated? I see Gaoler's Flail going off 0.3s prior to Fire/Light.

Reply:
Damage seems to snapshot at the same time, or at least I've not seen anyone successfully dodge one but not the other.

It would also significantly clutter the timeline making seeing upcoming mechanics harder.

Reply:
I think in this case, adding two extra lines wouldn't make it that much harder to see things coming ahead.  You could still see that a tankbuster was coming thirty seconds ahead with 5 lines.  Am I misunderstanding the situation?

I think in general unless there's a good reason, timelines should be single ability per line.  My subjective reasoning here is:
* readability (long lines are harder, for me at least)
* testing (you've dropped some ability lines here, so no longer verifying they're occurring)
* consistency (this is what most other timelines do)
* translation ease (don't have to translate another combo ability, can just have the Akurosia automatic ability translation cover this)

Reply:
I have no issues splitting these up, however checking across multiple pulls, it's dependent on which variation of flail if the snapshot is exactly the same timestamp or if it's offset by 0.3s. Should these just always use the same timestamp or how should it be handled?

Reply:
Are you saying that at a particular timestamp there's sometimes a delay and sometimes not, or that the delay is different at timestamp A and B, but consistent within A and consistent within B?

Reply:
The correct timestamp B depends on the ability for timestamp A which is random.

I split them apart with the latest commit, just using the same timestamps. Can be adjusted later if needed I guess?

Reply:
Sounds good! Thank you!